NineAnimator/Models/Anime Source/animeunity.it/AnimeUnity+Anime.swift

```diff
@@ -21,91 +21,85 @@ import Foundation
 import SwiftSoup
 
 extension NASourceAnimeUnity {
+    struct SearchResponseRecordsAnime: Codable {
+        var number: String = ""
+        var link: String = ""
+    }
+    
+    private struct DummyCodableAnime: Codable {}
+    
+    struct SearchResponseAnime: Codable {
+        var to_array: [SearchResponseRecordsAnime]
+        init(from decoder: Decoder) throws {
+            var to_array = [SearchResponseRecordsAnime]()
+            var container = try decoder.unkeyedContainer()
+            while !container.isAtEnd {
+                if let route = try? container.decode(SearchResponseRecordsAnime.self) {
+                    to_array.append(route)
+                } else {
+                    _ = try? container.decode(DummyCodableAnime.self) // <-- TRICK
```

Comment:
Why is this necessary? Can you send me an example response that you'll have to parse with this code?

Reply:
I have found a better way to do it, I'll push an update